pkg/config/setup/system_probe.go

```diff
@@ -246,6 +247,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnv(join(netNS, "max_http_stats_buffered"), "DD_SYSTEM_PROBE_NETWORK_MAX_HTTP_STATS_BUFFERED")
 	cfg.BindEnv(join(smNS, "max_http_stats_buffered"))
 	cfg.BindEnvAndSetDefault(join(smNS, "max_kafka_stats_buffered"), 100000)
+	cfg.BindEnv(join(smNS, "max_postgres_stats_buffered"))
```

Comment:
Same question as above for here, I saw that you used the cmd/system-probe/config/adjust_usm.go to set the value. But I am wondering what is the correct way to do so, currently seems like we do both

Reply:
same as https://github.com/DataDog/datadog-agent/pull/25416#discussion_r1592409582